### ISSUE TITLE
[Package Request]: Subsembly - Banking4 (help needed with silent switch)

### ISSUE THREAD

**Issue by borque** [microsoft/winget-pkgs#82443]
### Package Requested

- [ ] I would like someone else to build the manifest.
- [X] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not contained in a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

**Publisher:**
Subsembly
**Publisher Url:**
https://subsembly.com/index.html
**Package Name:**
Banking4
**Description:**
A software for onlinebanking and organization of bank accounts, mostly in Germany. It supports FinTS, HBCI and for some banks screen scraping to.
**Package Version:**
7.9.1.8280
**Installer URL:**
https://subsembly.com/download/TopBanking4Setup.exe

I think I could build that package, but I am unable to find a silent switch for the installer. Neither by USSF, nor by ProcessExplorer or testing things like / (or-) [s]ilent, [q]uiet, [u]natended, ...
WinRAR reports the exe-installer to be a self extracting cab (an can open it) in case that helps.

**Comment by Trenly** [comment]
I don't believe it has a silent switch. Looking inside the executable it has an ini file which indicates it was put together with PortableApps.com and everything I could find indicates that no silent switch is available using their install technology

**Comment by SpecterShell** [comment]
There are two kinds of switches available: `UPDATE ` and `UPDATE USB `. Note that there is a white space suffix at the end of each switch.
However when I use `UPDATE ` it shows this:
![image](https://user-images.githubusercontent.com/56779163/195498714-ca40428c-6378-4895-9909-069f57084a5c.png)

**Comment by Trenly** [comment]
@msftbot interactive-only

**Comment by dmacvicar** [comment]
If it is useful to you, when packaging this for nix/wine, I ran into the same problem, but at the end realized the app is portable and self-contained into a directory, and you can unpack the .exe with cabextract.
